Makefile

```diff
@@ -98,3 +105,18 @@ pipelines: ## Upload pipelines to Concourse
 showenv: ## Display environment information
 	$(eval export TARGET_CONCOURSE=deployer)
 	@concourse/scripts/environment.sh
+
+.PHONY: manually_upload_certs
+manually_upload_certs: ## Display environment information
+	@if ! aws s3 ls s3://${DEPLOY_ENV}-state/cf-certs.tfstate --summarize | grep -q "Total Objects: 0"; then \
+		aws s3 cp s3://${DEPLOY_ENV}-state/cf-certs.tfstate cf-certs.tfstate; \
+	else \
+		echo "No previous cf-certs.tfstate file found in s3://${DEPLOY_ENV}-state/. Assuming first run."; \
+	fi
+	@terraform apply -var env=${DEPLOY_ENV} \
+		-var-file=terraform/${AWS_ACCOUNT}.tfvars \
+		-state=cf-certs.tfstate \
+		-var router_external_crt="$$(PASSWORD_STORE_DIR=~/.paas-pass pass certs/${AWS_ACCOUNT}/${DEPLOY_ENV}/router_external.crt)" \
+		-var router_external_key="$$(PASSWORD_STORE_DIR=~/.paas-pass pass certs/${AWS_ACCOUNT}/${DEPLOY_ENV}/router_external.key)" \
```

Comment:
This password store setup isn't correct for the credentials-high repo. It might be better for this task to require `PASSWORD_STORE_DIR` to be set manually.